Atos20/indecision-maker-server
# Movie setup bd

### PR Template:
#### What's this PR do?

- Adds migration schema for the movies table
- Fetches 100 movies from IMDB API (currently hardcoded for family genre)
- While loop to seed 25 individual movies based on current genre into database
- Easily scalable and dynamic structure if we choose to implement more movies or different genres

#### Where should the reviewer start?  

- Start by reviewing logic and error handling for bad data, check if anything can be cleaned up

#### How should this be manually tested?  
- Pull changes locally 
- run `knex migrate:latest` `knex seed:run` and wait roughly 20 seconds for data to populate
- check that 25 unique family movies are added to the movies table

#### Any background context you want to provide?  
- Some inconsistent data in regard to ratings (not provided, different regions, etc..)
- Still don't have a path for actually watching the movie

#### What are the relevant tickets?  

closes #11 

#### Screenshots (if appropriate) 

![Screen Shot 2020-10-28 at 9 03 22 PM](https://user-images.githubusercontent.com/65369275/97512872-04eb8c80-1961-11eb-9e1a-50edf9c7e0f0.png)
